ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -0,0 +1,192 @@
+#include <wiredtiger.h>
+#include <wiredtiger_ext.h>
+#include <fstream>
+#include <list>
+#include <errno.h>
+#include <filesystem>
+#include <mutex>
+
+#include "gcp_connection.h"
+#include "gcp_log_system.h"
```

Comment:
we can remove this since it doesn't exist.

Reply:
I removed #include "gcp_connection.h"